rcl/test/rcl/test_time.cpp

```diff
@@ -730,3 +774,29 @@ TEST(CLASSNAME(rcl_time, RMW_IMPLEMENTATION), add_remove_add_jump_callback) {
     rcl_get_error_string().str;
   EXPECT_EQ(1u, clock->num_jump_callbacks);
 }
+
+TEST(CLASSNAME(rcl_time, RMW_IMPLEMENTATION), failed_get_now) {
+  rcl_allocator_t allocator = rcl_get_default_allocator();
+  rcl_clock_t uninitialized_clock;
+  rcl_time_point_value_t query_now;
+  rcl_ret_t ret = rcl_clock_init(RCL_CLOCK_UNINITIALIZED, &uninitialized_clock, &allocator);
```

Comment:
@Blast545 no `rcl_clock_fini()` counterpart?

Reply:
The function `rcl_clock_fini()` is not expected to be used on `RCL_CLOCK_UNINITIALIZED` clocks, it returns `RCL_RET_INVALID_ARGUMENT`